app/models/case/sar/standard.rb

```diff
@@ -146,11 +151,20 @@ def reset_deadline!
     )
   end
 
-  # SARs extensions are based on calendar days not working days
+  # The deadlines are all calculated based on the date receiving the case
```

Comment:
Wondering if this could be phrased as 
```suggestion
  # The deadlines are all calculated based on the date the case is received
```